tests/test_zogy.py

```diff
@@ -259,6 +268,37 @@ def testFourierTransformConvention(self):
             msg="Numpy FFT does not use expected default normalization"
             " convention (1 in forward, 1/Npix in inverse operation).")
 
+    def testSplitBorder(self):
+        """Test outer border box splitting around an inner box"""
+        config = ZogyConfig()
+        task = ZogyTask(config=config)
+
+        bb = geom.Box2I(geom.Point2I(5, 10), geom.Extent2I(20, 30))
+        D = afwImage.ImageI(bb)
+        innerbox = bb.erodedBy(geom.Extent2I(3, 4))
+        D[innerbox] = 1
+
+        borderboxes = task.splitBorder(innerbox, bb)
+        for x in borderboxes:
+            D[x] += 1
+        # The splitting should cover all border pixels exactly once
+        self.assertTrue(np.all(D.array == 1), "Border does not cover all pixels exactly once.")
+
+    def testGenerateGrid(self):
```

Comment:
Is there possibly a need to test the case where the "grid" has only one cell? That is the full exposures are used and not broken up?

Reply:
Added.